opensoundscape/localization.py

```diff
@@ -68,38 +58,38 @@ def __init__(
         self.class_name = class_name
         self.bandpass_range = bandpass_range
         self.cc_threshold = cc_threshold
+        self.cc_filter = cc_filter
         self.max_delay = max_delay
 
-        # check that max_delay is not longer than the duration of the audio and raise a value error if it is
+        # Verify that max_delay is not longer than the duration of the audio and raise a value error if it is
         if self.max_delay >= self.duration:
             raise ValueError(
                 f"max_delay ({self.max_delay}) is longer than duration ({self.duration}) of audio clips."
             )
-        # initialize attributes to store values calculated by methods
+
+        # Initialize attributes. These will be set by the methods estimate_delays() and estimate_location()
         self.tdoas = None  # time delay at each receiver
         self.cc_maxs = None  # max of cross correlation for each time delay
         self.location_estimate = None  # cartesian location estimate in meters
+        self.distance_residuals = None  # distance residuals in meters
+        self.receivers_used_for_localization = (
+            None  # list of receivers used for localization
+        )
         self.residual_rms = None
 
-        # could implement this later:
-        # hidden attributes store estimates and error metrics
-        # from gillette and soundfinder localization algorithms
-        # self._gillette_location_estimate = None
-        # self._gillette_error = None
-        # self._soundfinder_location_estimate = None
-        # self._soundfinder_pseudorange_error = None
-
     def estimate_location(
         self,
         algorithm="gillette",
-        cc_threshold=None,
+        cc_filter="phat",
         min_n_receivers=3,
+        cc_threshold="default",
```

Comment:
same comment as below about None vs "default" for both of these arguments

Reply:
See other comment. I want to allow the user to either:
1. Specify a bandpass range different to that stored in the object's attributes.
2. Use the default bandpass range, stored in the object's attributes.
3. Specify NO bandpassing.